### PR TITLE
update hugging face runtime documentation

### PR DESCRIPTION
**Pull request by mohittalele**
I recently implemented a custom runtime for hugging face sentence transformer. I updated documentation so make clear which model are supported and where one should use custom model when it comes to hugging face models.

### CODE REVIEW

**Review by edshee — APPROVED** [review]
Good addition, thanks Mohit

File: docs/examples/huggingface/README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ We will show how to add share a task
     }
 }
 ```
+The task section in the huggingface runtime refers to the hugginface piplines for each of their predefined tasks. You can refer                           [huggingface documentation](https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.pipeline.task) for updated list of out of box supported models. The model listed in "task" section above needs to support one of their base "tasks". If thats not the case then you need to wrap around your model using custom runtime.
```
Comment (adriangonz) [inline docs/examples/huggingface/README.md:38]: typo: `pipelines` instead of `piplines`

**Review by adriangonz — COMMENTED** [review]
Nice one @mohittalele! Thanks for adding that clarification. 

There's a small typo, but everything else looks good to go :+1:

**Comment by adriangonz** [comment]
Hey @mohittalele ,

I think you've pushed a couple extra commits to the PR by mistake. If you can revert those, (and fix the typo on the original docs changes) - this should be good to go.

**Comment by mohittalele** [comment]
Hey @adriangonz 
yes, those commit were by mistake. Some unexpected stuff happened. created a same PR - #1267

**Pull request closed without merging**